metrics.yml

```diff
@@ -0,0 +1,6 @@
+---
+component: side-indexer-lambda
+interval: 1000
+metrics:
+  s3-reads: AWS S3 GetObject
+  s3-writes: AWS S3 PutObject
```

Comment:
I'm not 100% know what this is for or if it works...

Reply:
This demands some extra logic with OpenTelemetry for working. If you want to keep it simple (w/o metrics) remove it here and also from the Dockerfile Copy command